clang/test/Driver/sycl-device-lib-amdgcn.cpp

```diff
@@ -0,0 +1,45 @@
+// Tests specific to `-fsycl-targets=amdgcn-amd-amdhsa`
+// Verify that the correct devicelib linking actions are spawned by the driver.
+// Check also if the correct warnings are generated.
+
+// UNSUPPORTED: system-windows
+// REQUIRES: amdgpu-registered-target
```

Comment:
The use of a `REQUIRES` target shouldn't be necessary with driver specific tests.  The use of the `--sysroot=%S/Inputs/SYCL` will allow for local dummy files to be picked up to verify driver functionality.  Adding an empty `Inputs/SYCL/lib/devicelib--amd.bc` file should help this along.

Reply:
Ah, I see. Thanks for clarifying this.
I've removed this and added a dummy input